.rubocop.yml

```diff
@@ -7,6 +7,7 @@ AllCops:
     - Rakefile
     - Gemfile
     - Capfile
+    - spec/**/*
```

Comment:
Is it really required here?

Reply:
The indents and good code look is checked in this way. If we potentially do not need it, I can remove it from rubocop configs.